packages/server/__snapshots__/6_uncaught_support_file_spec.coffee.js

```diff
@@ -22,16 +22,17 @@ exports['e2e uncaught support file errors failing 1'] = `
   1 failing
 
   1)  An uncaught error was detected outside of a test:
-     Uncaught Error: bar
+     Uncaught Error: The following error originated from your test code, not from Cypress.
 
-This error originated from your test code, not from Cypress.
+  > bar
 
 When Cypress detects uncaught errors originating from your test code it will automatically fail the current test.
 
 Cypress could not associate this error to any specific test.
 
 We dynamically generated a new test to display this failure.
-  
+  Error: bar
```

Comment:
Another case of the original message not getting properly replaced in the stack.